src/main/js/PullRequestInformation.tsx

```diff
@@ -57,135 +55,121 @@ export function isUrlSuffixMatching(baseURL: string, url: string, suffix: string
   return strippedUrl === suffix;
 }
 
-class PullRequestInformation extends React.Component<Props> {
-  static defaultProps = {
-    shouldFetchChangesets: true
-  };
+const PullRequestInformation: FC<Props> = ({
+  repository,
+  shouldFetchChangesets = true,
+  pullRequest,
+  mergeHasNoConflict,
+  source,
+  target,
+  targetBranchDeleted
+}) => {
```

Comment:
`status` is defined in props type and is used in function body, but it is not spreaded from the props.